### PR TITLE
Refactoring comment docs of `Application`.

### PR DESCRIPTION
**Pull request by mdmunir**
#10947

Before refactoring:
screenshot from 2016-02-24 14 13 52

After:
screenshot from 2016-02-24 14 06 25

screenshot from 2016-02-24 14 07 41

### CODE REVIEW

**Comment by mdmunir** [comment]
> PS: When i do `composer install` i got
screenshot from 2016-02-24 15 42 35

File: framework/base/Application.php [hunk 1]
```diff
@@ -20,15 +20,10 @@
  * @property \yii\caching\Cache $cache The cache application component. Null if the component is not enabled.
  * This property is read-only.
  * @property \yii\db\Connection $db The database connection. This property is read-only.
- * @property \yii\web\ErrorHandler|\yii\console\ErrorHandler $errorHandler The error handler application
```
Comment (githubjeka) [inline framework/base/Application.php]: Сunning fix , but you should not remove these properties from here.

**Comment by mdmunir** [comment]
@githubjeka 
They are no `$errorHandler`, `$request` and `$response` in `yii\base\Application::coreComponents()`
https://github.com/yiisoft/yii2/blob/master/framework/base/Application.php#L619-L626

But its ok, i can revert it

**Comment by cebe** [comment]
closing as this solution does not work. see https://github.com/yiisoft/yii2/issues/10947#issuecomment-188673545

**Pull request closed without merging**